src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Responses/RoatpV2/GetProviderSummaryResponse.cs

```diff
@@ -1,63 +1,5 @@
-namespace SFA.DAS.SharedOuterApi.InnerApi.Responses.RoatpV2
+namespace SFA.DAS.SharedOuterApi.InnerApi.Responses.RoatpV2;
+
+public class GetProviderSummaryResponse : Provider
 {
-    public class GetProviderSummaryResponse
```

Comment:
Delete commented code

Reply:
it's all deleted, not sure what you mean